src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -83,6 +83,52 @@ the MIT License. See LICENSE in the project root for license information. -->
                                  Text="{x:Bind State.Profile.TabTitle, Mode=TwoWay}"
                                  Style="{StaticResource TextBoxSettingStyle}"/>
                     </ContentPresenter>
+
+                    <!--Delete Button-->
+                    <Button Click="Delete_Click"
+                            Style="{StaticResource BaseButtonStyle}"
+                            Margin="{StaticResource StandardControlSpacing}"
+                            IsEnabled="{x:Bind State.CanDeleteProfile}">
+                        <Button.Resources>

```

Comment:
Should the styling for these buttons be in `CommonResources.xaml`? I'm _presuming_ they're styled the same?

Reply:
Creating a new `DeleteButtonStyle`. _Unfortunately_, I can't add the theming, font icon, and layout... I would have to retemplate it, I think? So really I just added the margin to the BaseButtonStyle hahaha.